Run/CMakeLists.txt

```diff
@@ -4,4 +4,4 @@
 #
 
 add_run(EnergyPositionHistos EnergyPositionHistosRun.cpp)
-add_run(TruthTracking TruthTrackingRun.cpp)
+#add_run(TruthTracking TruthTrackingRun.cpp)
```

Comment:
The better cmake options are merged right now. I believe, there's no need to comment that out